streampipes-client-python/pyproject.toml

```diff
@@ -60,6 +60,7 @@ pandas = ">=1.5, <3"
 pydantic = "~2, >=2.6.3"
 requests = "~2, >=2.28"
 typing-extensions = "~4, >=4.5"
+mkdocs-git-revision-date-localized-plugin = "^1.2.6"
```

Comment:
Can we please add it to the docs group and pin the version?

Reply:
Okay I fixed that.